test/src/test/java/org/corfudb/infrastructure/BaseServerTest.java

```diff
@@ -13,31 +13,33 @@
  */
 public class BaseServerTest extends AbstractServerTest {
 
-    BaseServer server;
+    BaseServer bs = new BaseServer();
```

Comment:
Seems like initialization may not actually be needed here, since @Before already does it?